TAScheduler/Managment/UserManagement.py

```diff
@@ -26,24 +24,15 @@ def createUser(self, Id, name, contact, address, password, email,usertype):
         if(not(isinstance(address, str))):
             raise TypeError("Address entered is not of type str")
         if(not(isinstance(password, str))):
-            raise TypeError("Address entered is not of type str")
-
-        try:
-            self.findUser(Id)
-        except TypeError: 
-            UserProfile.objects.create(
-                userID = Id,
-                userType = usertype,
-                userPassword = password,
-                userName = name,
-                userAddress = address,
-                userContact = contact,
-                userEmail = email
-            )
-            return
-
-        raise TypeError("This user already exists: ")
+            raise TypeError("Password entered is not of type str")
+        if(usertype != "TA" or usertype != "SUPERVISOR" or usertype != "INSTRUCTOR"):
```

Comment:
```suggestion
        if not(userType in ["SUPERVISOR", "INSTRUCTOR", "TA"]):
```
Slightly more readable.